.github/workflows/test.yml

```diff
@@ -503,16 +503,17 @@ jobs:
     needs: setup
     environment:
       name: ${{ needs.setup.outputs.environment }}
-    env:
-      GEF_DEAL_VERSION: 0
     runs-on: ubuntu-latest
 
     strategy:
       # Do not cancel in-progress jobs upon failure
       fail-fast: false
+      # Single dimension matrix
+      matrix:
+        spec: ['facility/**/*.spec.js']
```

Comment:
Decided its probably worthwhile just testing all the facility e2e tests to make sure I dont miss something